workflow-examples/src/main/java/com/redhat/parodos/examples/simple/RestAPIWorkFlowTask.java

```diff
@@ -41,7 +40,7 @@
 @Slf4j
 public class RestAPIWorkFlowTask extends BaseInfrastructureWorkFlowTask {
 
-	private static final String PAYLOAD_PASSED_IN_FROM_SERVICE = "PAYLOAD_PASSED_IN_FROM_SERVICE";
+	static final String PAYLOAD_PASSED_IN_FROM_SERVICE = "PAYLOAD_PASSED_IN_FROM_SERVICE";
```

Comment:
same here?

Reply:
Similar as before, I use this constant in [RestAPIWorkFlowTaskTest](https://github.com/redhat-developer/parodos/blob/92236853d9327c02a3ed096032bcf743c9062e8e/workflow-examples/src/test/java/com/redhat/parodos/examples/simple/RestAPIWorkFlowTaskTest.java#L100). I can use another private string constant in the test if you think it's better.

Reply:
ack

Reply:
Used a private string constant in the test.